modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectActionLocalServiceTest.java

```diff
@@ -439,6 +440,18 @@ public void testAddObjectAction() throws Exception {
 				RandomTestUtil.randomString(), name,
 				ObjectActionTriggerConstants.KEY_ON_AFTER_DELETE, false));
 
+		AssertUtils.assertFailure(
+			ObjectActionDescriptionException.class,
+			"Description must be less than 75 characters",
+			() -> _addObjectAction(
+				RandomTestUtil.randomString(76),
+				RandomTestUtil.randomString(),
+				RandomTestUtil.randomString(),
+				RandomTestUtil.randomString(),
+				RandomTestUtil.randomString(),
+				ObjectActionTriggerConstants.KEY_ON_AFTER_UPDATE,
+				false));
```

Comment:
@Aquiiiles you don't need to create a new method that it's only used once. Please simplify:

```java
AssertUtils.assertFailure(
	ObjectActionDescriptionException.class,
	"Description must be less than 75 characters",
	() ->
		_objectActionLocalService.addObjectAction(
			RandomTestUtil.randomString(), TestPropsValues.getUserId(),
			_objectDefinition.getObjectDefinitionId(), true,
			RandomTestUtil.randomString(), RandomTestUtil.randomString(76),
			LocalizedMapUtil.getLocalizedMap(RandomTestUtil.randomString()),
			LocalizedMapUtil.getLocalizedMap(RandomTestUtil.randomString()), RandomTestUtil.randomString(),
			ObjectActionExecutorConstants.KEY_GROOVY, ObjectActionTriggerConstants.KEY_ON_AFTER_UPDATE,
			new UnicodeProperties(), false));
```